docs/querying/functions.md

```diff
@@ -617,7 +617,7 @@ Like `sort`, `sort_desc` only affects the results of instant queries, as range q
 
 ## `sort_by_label()`
 
-**This function has to be enabled via the [feature flag](../feature_flags/) `--enable-feature=promql-experimental-functions`.**
+**This function has to be enabled via the [feature flag](../feature_flags.md) `--enable-feature=promql-experimental-functions`.**
```

Comment:
We could even add the exact entry (`../feature_flags.md/#experimental-promql-functions`), but I think this is sufficient and future proof.

Reply:
Thank you for the review! Good idea. I followed up on this in https://github.com/prometheus/prometheus/pull/14630